browser/src/map/handler/Map.Keyboard.js

```diff
@@ -646,5 +646,14 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 		return false;
+	},
+
+	readOnlyAllowedShortcuts: function(e) {
+		if (this._isCtrlKey(e) && e.shiftKey && e.key === '?')
+			return true;
+		else if (e.type === 'keydown' && !e.shiftKey && !e.ctrlKey && !e.altKey && !e.metaKey && e.keyCode === 112)
```

Comment:
we have constants for keyCodes now: https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.Keyboard.js#L220

Also it would be good to add a comment what this condition detects :)
Is this for help? (F1 without special keys like ctrl)